### PR TITLE
Add support for generating a fake OCSP response

### PR DESCRIPTION
**Pull request by alin04**
Add support for generating a fake OCSP response to seed the cache in development mode.

### CODE REVIEW

**Comment by alin04** [comment]
I debated on replacing the fake ca.cert entirely, and regenerating the server.cert. Let me know your thoughts.

File: testdata/b1/README.md [hunk 1]
```diff
@@ -14,6 +14,28 @@ $ openssl x509 -req -in server.csr -CA ca.cert -CAkey ca.privkey -CAcreateserial
 $ cat server.cert ca.cert >fullchain.cert
 ```
 
+OCSP:
+
+Make sure your openssl.cfg has an ocsp extension section, like:
+
+[ ocsp ]
```
Comment (twifkak) [inline testdata/b1/README.md]: Put this and other code snippets in ``` quotes ``` like above. (Click View button to preview the markdown rendering.)

**Comment by twifkak** [comment]
Yeah, if this new ca cert can be used in the same way as the old one, then go ahead and replace and regenerate. Would be nice to keep it simple if possible. Note you'll have to fix some unit tests because they hard-code the cert hash.

**Comment by twifkak** [comment]
(OTOH LMK if there's some other reason not to replace & regenerate that you thought of.)

File: testdata/b1/README.md (hunk 1 above)
Comment (alin04) [reply testdata/b1/README.md]: Done

**Comment by alin04** [comment]
I kept the ocsp cert separate due to problems when I tried to set the OCSPSigner extension on the root cert.

**Pull request merged**